mos-platform/cpm65/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+platform(cpm65 HOSTED PARENT common)
+
+if(NOT CMAKE_CROSSCOMPILING)
+  return()
+endif()
+
+install(FILES
+  cpm.h
+TYPE INCLUDE)
+install(FILES link.ld TYPE LIB)
+
+add_platform_library(cpm65-crt0)
+merge_libraries(cpm65-crt0
+  common-init-stack
+  common-copy-zp-data
+  common-zero-bss
+)
+
+add_platform_library(cpm65-c
+  cpm.S
+  cpm-wrappers.c
+  bios.S
+  comhdr.S
+  pblock.S
+  putchar.c
+  stack.S
+  registers.S
+)
+merge_libraries(cpm65-c
```

Comment:
This should be automatically done by the add_platform_library directive. See if it still works with the `merge_libraries` removed.